lib/Constants.vala

```diff
@@ -285,6 +286,12 @@ namespace Granite {
          * Style class for non-terminal text that uses a monospace font.
          */
         public const string MONOSPACE = "monospace";
+
+        /**
+         * Internal style class for {@link Granite.TerminalView} to emulate the appearance of Terminal. This includes
+         * text color, background color, selection highlighting, and selecting the system monospace font.
+         */
+        internal const string TERMINAL = "terminal";
```

Comment:
I don't think this should be added as a constant here. We usually don't make constants for things that aren't used outside of their own class